packages/os/write-network-status.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Write network status
+# Block manual interactions with this service, since it could leave the system in an
+# unexpected state
+RefuseManualStart=true
+RefuseManualStop=true
+Before=set-hostname.service
+After=network-online.target systemd-resolved.service
+Wants=network-online.target systemd-resolved.service
```

Comment:
This ends up running concurrently with `early-boot-config` and - if it took longer to finish - might not be complete before settings generators ran.

```
[    4.228716] localhost systemd[1]: Starting Bottlerocket userdata configuration system...
[    4.230046] localhost systemd[1]: Starting Write network status...
[    4.240167] localhost early-boot-config[1042]: 15:38:52 [INFO] early-boot-config started
[    4.240167] localhost early-boot-config[1042]: 15:38:52 [INFO] Retrieving platform-specific data
[    4.251987] localhost early-boot-config[1042]: 15:38:52 [WARN] No user data found via local file: /var/lib/bottlerocket/user-data.toml
[    4.260923] localhost early-boot-config[1042]: 15:38:52 [INFO] Received dynamic/instance-identity/document
[    4.261748] localhost early-boot-config[1042]: 15:38:52 [WARN] No user data found.
[    4.261748] localhost early-boot-config[1042]: 15:38:52 [INFO] Sending instance identity document to API
[    4.263129] localhost systemd[1]: Finished Write network status.
[    4.266859] localhost systemd[1]: Finished Bottlerocket userdata configuration system.
[    4.311108] localhost systemd[1]: Starting User-specified setting generators...
```

Reply:
I'll add a `Before=early-boot-config.service` to avoid this.